<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BotTargetList</class>
 <widget class="QDialog" name="BotTargetList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bot and target</string>
  </property>
  <property name="windowIcon">
   <iconset resource="localr.qrc">
    <normaloff>:/6.png</normaloff>:/6.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBoxBot">
         <property name="maximumSize">
          <size>
           <width>230</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Bot</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QListWidget" name="bots">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBoxStep">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Step: Raw block</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Step: Grouped block</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Step: Simplified block (not coded)</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="browseMap">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="icon">
            <iconset resource="localr.qrc">
             <normaloff>:/6.png</normaloff>:/6.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="label_local_target">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Local target</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_select_map"/>
          </item>
          <item>
           <widget class="QCheckBox" name="trackThePlayer">
            <property name="text">
             <string>Track the player</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="mapPreview">
            <property name="font">
             <font>
              <pointsize>6</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/images/empty.png);</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>16</number>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>16</number>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>16</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>16</number>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPlainTextEdit" name="graphvizText">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="graphvizTextShow">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Layer">
            <item>
             <property name="text">
              <string>Layer X</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="localTargets">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>200</height>
             </size>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_zone">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_next_local_target">
            <property name="text">
             <string>Next local target: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="bestTarget">
            <property name="text">
             <string>Best target:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxglobalTarget">
       <property name="title">
        <string>Global target</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>All bots</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="autoSelectTarget">
             <property name="text">
              <string>Auto-select</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="autoSelectFilter">
             <property name="text">
              <string>Filter</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="globalTargets">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="hideTooHard">
          <property name="text">
           <string>Hide too hard item</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Deep of the search:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="searchDeep">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QPlainTextEdit" name="overall_graphvizText">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="overall_graphvizTextShow">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxPlayer">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Player</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_player_cash">
          <property name="text">
           <string>Cash: 0$</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Inventory</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QListWidget" name="inventory">
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxFight">
             <property name="title">
              <string>Fight</string>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Hp other monster</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QProgressBar" name="fightOtherHp"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Monster</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="monsterFightName"/>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="monsterFightImage"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Monster</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QListWidget" name="monsterList">
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Preference</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Plant</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="PrefPlant">
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="sliderPosition">
                   <number>50</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Item</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="PrefItem">
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="sliderPosition">
                   <number>50</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Fight</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="PrefFight">
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="sliderPosition">
                   <number>50</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Shop</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="PrefShop">
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="sliderPosition">
                   <number>50</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Wild</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="PrefWild">
                  <property name="maximum">
                   <number>100</number>
                  </property>
                  <property name="sliderPosition">
                   <number>50</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_action">
     <property name="text">
      <string>Other info</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="localr.qrc"/>
 </resources>
 <connections/>
</ui>
